NangPaGo-be/src/main/java/com/mars/NangPaGo/config/SecurityConfig.java

```diff
@@ -54,7 +54,8 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                     "/oauth2/**",
                     "/auth/reissue",
                     "/auth/status",
-                    "/common/example/**"  // TODO: 제거 예정
+                    "/recipe/**",
```

Comment:
레시피 좋아요, 즐겨찾기 기능이 `/recipe/toggle/like`, `/recipe/toggle/favorite` 과 같은 형식으로 추가될 예정이라, 와일드카드 처리는 하면 안될 것 같아요.

```suggestion
                    "/recipe",
                    "/recipe/{id}",
```

Reply:
넵